framework/db/Command.php

```diff
@@ -91,7 +91,7 @@ class Command extends Component
     /**
      * @var array pending parameters to be bound to the current PDO statement.
      */
-    private $_pendingParams = [];
+    protected $_pendingParams = [];
```

Comment:
- Protected properties do not use `_` in the name.
- `@since` annotation should be added because it would be part of API.